_includes/schemas/trackers_response.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "the tracker id that can be used for requesting info about a tracker"
+    },
+    "carrier_tracking_no": {
```

Comment:
This is returned as `tracking_code` at the moment. This is a bit confusing, since it is called `carrier_tracking_no` when creating the Tracker, which is probably a bug? We should address this at some point :wink: